### ISSUE TITLE
Cannot run liferay-npm-scripts format for out of src tests

### ISSUE THREAD

**Issue by andresfulla** [liferay/liferay-npm-tools#27]
I had to run `npx csf {src,test}/**/*.es.js src/**/*.scss` to get my tests formatted. I am not able to pass that config to `liferay-npm-scripts format`, and the script itself does not read `/test/` folder.

**Comment by bryceosterhaus** [comment]
Hey @andresfulla, how exactly were you trying to passing it to `liferay-npm-scripts`?

Could you link me to the specific project?

**Comment by bryceosterhaus** [comment]
Relates, https://github.com/liferay/liferay-npm-tools/pull/29#discussion_r262584950

**Comment by bryceosterhaus** [comment]
@andresfulla  Is this still an issue for you? If so, can you link me to your project and the config?

**Comment by wincent** [comment]
You should be able to set this up with an `npmscripts.config.js` file (something) like this in the project root:

```javascript
const standardPreset = require('liferay-npm-scripts/src/presets/standard/index');

module.exports = {
	preset: 'liferay-npm-scripts/src/presets/standard/index',
	lint: [
		...standardPreset.lint, 
		'src/**/*.scss',
		'src/**/*.es.js',
		'test/**/*.es.js',
	]
};
```

Please let us know @andresfulla if that doesn't work for you.

In any case, we should probably audit the way different modules in liferay-portal are laying out their tests, make them all look the same, and then bake it into the default config.

**Comment by andresfulla** [comment]
@bryceosterhaus sorry for the late response. _(I'm working on my Github notifications management abilities at this very moment)_. This is an issue for us in the [`segments/segments-web` module](https://github.com/liferay/liferay-portal/tree/master/modules/apps/segments/segments-web) in the portal.

We are at the moment using this two scripts to format our code:
```
...
"csf": "csf {src,test}/**/*.es.js src/**/*.scss",
"format": "npm run csf -- -i",
...
```

@wincent while your solution does work, I'm not confident because I'm not sure if I'm overwriting the whole config object or composing on top of the default values.

I'd like the `csf <files path>` in `liferay-npm-scripts format` just in case I have different formatting needs.

Alternately, including `test/**/*.es.js` files as default files to look at will do the trick for me, and it's logical as test files need to be formatted and that's one of the places where `liferay-npm-scripts test` expects them to be.

Hope this make sense. Thanks!

**Comment by wincent** [comment]
@andresfulla:

> I'm not sure if I'm overwriting the whole config object or composing on top of the default values.

You are effectively "overwriting" it. The "preset" entry says to use the standard default as a starting point, and you then completely replace the "lint" config with the one that I showed above. Note that we spread in the `...standardPreset.lint` config though and then add a few entries, so our "overwrite" really consists of adding some things to the end.

If we're going to add to the default, can you think of any reason why we wouldn't just add `test/**/*.js` (instead of `test/**/*.es.js`)?

**Comment by andresfulla** [comment]
I see. Understood. Thanks

I don't fully understand why `.es.js` files are suffixed that way, so, no, I can't find a reason. In our module there's nothing that could go wrong adding `test/**/*.js` instead of `test/**/*.es.js`.

**Comment by jbalsas** [comment]
Just for context, we decided to suffix modules with `.es.js` (wish we would've come up with `.mjs` instead 😂)  back in 2015 when we added support for ES2015 in portal as a way to detect which files would need to be transpiled and wrapped in amd declarations.

To this day, we should probably still use the convention in modules that contain old modules. In fact, those are excluded in the babel config to prevent unnecessary modifications.
